saidump/saidump.cpp

```diff
@@ -143,11 +107,10 @@ CmdOptions handleCmdLine(int argc, char **argv)
                 exit(EXIT_FAILURE);
         }
     }
-
-    return options;
+    return status;
```

Comment:
this function does not need to return status, since status in this function is always success, and on any failure exit(1) is called, so ether make function void, or return status filure on error

Reply:
Fixed.